draft-duke-quic-version-aliasing.md

```diff
@@ -323,6 +323,9 @@ The Packet Length Offset is also encoded as a Variable Length Integer.
 Clients can determine the length of the Initial Token Extension by subtracting
 known and encoded field lengths from the overall transport parameter length.
 
+Client storage of this transport parameter is mandatory if it seeks to send
+0-RTT packets with the corresponding aliased version.
```

Comment:
This seems like a tautology, doesn't it?  The question you need to ask is whether a server will accept 0-RTT without an aliased version.

Reply:
Yeah, I'm just trying to follow the letter of quic-transport even though it's a little silly in this case. I'll add something about the server.